cmd/crates/soroban-test/tests/fixtures/test-wasms/udt/src/lib.rs

```diff
@@ -34,6 +34,7 @@ pub struct Contract;
 
 #[contractimpl]
 impl Contract {
+    #[allow(clippy::unnecessary_fold)]
```

Comment:
Still needed if we switch to sum?

Reply:
I thought it was a nice example for readers that they could do it both ways.